.github/workflows/lock.yaml

```diff
@@ -0,0 +1,24 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
```

Comment:
Is this a reliable and widely used plugin which we can trust and use?

Reply:
Used in below repos - 
- [Jest](https://github.com/facebook/jest/blob/master/.github/workflows/lock.yml) 
- [Flutter ](https://github.com/flutter/flutter/blob/master/.github/lock.yml)
- [Next.js](https://github.com/vercel/next.js/blob/canary/.github/lock.yml)